client-spark/common/src/main/java/org/apache/spark/shuffle/RssShuffleHandle.java

```diff
@@ -69,19 +58,15 @@ public ShuffleDependency<K, V, C> getDependency() {
     return dependency;
   }
 
-  public Map<Integer, List<ShuffleServerInfo>> getPartitionToServers() {
```

Comment:
Can we keep this method? In this way, our changes will be smaller.

Reply:
yes, it's better.